packages/micro.rb

```diff
@@ -25,4 +21,6 @@ class Micro < Package
   def self.install
     system "install -Dm755 micro #{CREW_DEST_PREFIX}/bin/micro"
   end
+
+  no_compile_needed
```

Comment:
Why the move down here?

Reply:
The normal layout of a package has 
```
# package information
source_url
version
binary_sha256

# package dependencies
depends_on
depends_on
depends_on

# package actions
def self.patch
  ...
end

def self.install
  ...
end

# package properties
no_lto
no_compile_needed
configure_options
```

I'll try and set up a rubocop cop to enforce this at some point

Reply:
Most of our packages have the boolean properties above the def functions.

Reply:
Ah, you're right. I mixed them up with buildsystem properties. Boolean properties above def functions, buildsystem properties below.